src/overlays/actors/ovl_En_GeldB/z_en_geldb.h

```diff
@@ -7,31 +7,31 @@
 struct EnGeldB;
 
 typedef enum {
-    /* 00 */ GELDB_LIMB_NONE,
-    /* 01 */ GELDB_LIMB_ROOT,
-    /* 02 */ GELDB_LIMB_TORSO,
-    /* 03 */ GELDB_LIMB_NECK,
-    /* 04 */ GELDB_LIMB_PONYTAIL,
-    /* 05 */ GELDB_LIMB_VEIL,
-    /* 06 */ GELDB_LIMB_HEAD,
-    /* 07 */ GELDB_LIMB_R_UPPER_ARM,
-    /* 08 */ GELDB_LIMB_R_FOREARM,
-    /* 09 */ GELDB_LIMB_R_WRIST,
-    /* 10 */ GELDB_LIMB_R_HAND,
-    /* 11 */ GELDB_LIMB_R_SWORD,
-    /* 12 */ GELDB_LIMB_L_UPPER_ARM,
-    /* 13 */ GELDB_LIMB_L_FOREARM,
-    /* 14 */ GELDB_LIMB_L_WRIST,
-    /* 15 */ GELDB_LIMB_L_HAND,
-    /* 16 */ GELDB_LIMB_L_SWORD,
-    /* 17 */ GELDB_LIMB_L_THIGH,
-    /* 18 */ GELDB_LIMB_L_SHIN,
-    /* 19 */ GELDB_LIMB_L_FOOT,
-    /* 20 */ GELDB_LIMB_R_THIGH,
-    /* 21 */ GELDB_LIMB_R_SHIN,
-    /* 22 */ GELDB_LIMB_R_FOOT,
-    /* 23 */ GELDB_LIMB_WAIST,
-    /* 24 */ GELDB_LIMB_MAX
+    /* 0x00 */ GELDB_LIMB_NONE,
+    /* 0x01 */ GELDB_LIMB_ROOT,
+    /* 0x02 */ GELDB_LIMB_TORSO,
+    /* 0x03 */ GELDB_LIMB_NECK,
+    /* 0x04 */ GELDB_LIMB_PONYTAIL,
+    /* 0x05 */ GELDB_LIMB_VEIL,
+    /* 0x06 */ GELDB_LIMB_HEAD,
+    /* 0x07 */ GELDB_LIMB_R_UPPER_ARM,
+    /* 0x08 */ GELDB_LIMB_R_FOREARM,
+    /* 0x09 */ GELDB_LIMB_R_WRIST,
+    /* 0x0A */ GELDB_LIMB_R_HAND,
+    /* 0x0B */ GELDB_LIMB_R_SWORD,
+    /* 0x0C */ GELDB_LIMB_L_UPPER_ARM,
+    /* 0x0D */ GELDB_LIMB_L_FOREARM,
+    /* 0x0E */ GELDB_LIMB_L_WRIST,
+    /* 0x0F */ GELDB_LIMB_L_HAND,
+    /* 0x10 */ GELDB_LIMB_L_SWORD,
+    /* 0x11 */ GELDB_LIMB_L_THIGH,
+    /* 0x12 */ GELDB_LIMB_L_SHIN,
+    /* 0x13 */ GELDB_LIMB_L_FOOT,
+    /* 0x14 */ GELDB_LIMB_R_THIGH,
+    /* 0x15 */ GELDB_LIMB_R_SHIN,
+    /* 0x16 */ GELDB_LIMB_R_FOOT,
+    /* 0x17 */ GELDB_LIMB_WAIST,
+    /* 0x18 */ GELDB_LIMB_MAX
```

Comment:
Limbs should decimal IMO but if someone said to use hex leave it.

Reply:
I asked in Discord and @Roman971 said all enums in hex if possible: https://discord.com/channels/688807550715560050/688851317593997489/828722876801613895